docs/administration/password-recovery.md

```diff
@@ -0,0 +1,16 @@
+# Password recovery by user
+
+If user has forgotten his password and wants to recover it on his own, he needs to do the following:
```

Comment:
If user => If a user.
his => their
on his own => better to remove the phrase
he => they